src/anim/anim.js

```diff
@@ -694,17 +694,26 @@ Object.assign(pc, function () {
                 if (!object ||
                     !object.model ||
                     !object.model.model ||
-                    !object.model.model.morphInstances ||
-                    !object.model.model.morphInstances[0]) {
+                    !object.model.model.morphInstances) {
+                    return null;
+                }
+                var meshInstances = object.model.model.meshInstances;
```

Comment:
Actually, you can just do `object.model.meshInstances` 😄  Wow, you can save 6 characters!

Reply:
Every little helps :)